pkgs/development/python-modules/orbax-checkpoint/default.nix

```
@@ -90,7 +90,7 @@ buildPythonPackage rec {
 
   meta = {
     description = "Orbax provides common utility libraries for JAX users";
-    homepage = "https://github.com/google/orbax/tree/main/checkpoint";
+    homepage = "https://github.com/google/orbax/blob/v${version}/checkpoint/CHANGELOG.md";
     changelog = "https://github.com/google/orbax/releases/tag/v${version}";
```

```suggestion
    homepage = "https://github.com/google/orbax/tree/main/checkpoint";
    changelog = "https://github.com/google/orbax/blob/${src.rev}/checkpoint/CHANGELOG.md";
```
or use `lib.removePrefix "refs/tags/" src.rev`

---

Done